PRESIGNED-PAUSE.md

```diff
@@ -40,6 +42,19 @@ doesn’t even need to be funded. These are offchain signatures.
 
 A Facilitator will collect the signatures and execute the contract.
 
+### 1. Ensure no GAPs with PSPs coverage
```

Comment:
nit, let's delete the number since it's the only header, and the other numbered headers are to indicate steps in a sequence (table of contents will also need to be updated)
```suggestion
### Ensure no GAPs with PSPs coverage
```